src/server/controllers/users.controller.js

```diff
@@ -92,6 +92,7 @@ export function signin(req, res, next) {
         res.status(400).json(err);
       } else {
         if (doc.length == 0) {
```

Comment:
I think we need to start using ===.
https://github.com/airbnb/javascript#comparison--eqeqeq